package.json

```
@@ -139,7 +139,7 @@
     "parse-dashboard": "./bin/parse-dashboard"
   },
   "engines": {
-    "node": ">=18.0.0 <21"
+    "node": ">=18.0.0 <21||22"
```

The min supported node 22 version is 22.9.0

```suggestion
    "node": ">=18.0.0 <21 || >=22.9.0 <23"
```